extensions/oidc-client-registration/runtime/src/main/java/io/quarkus/oidc/client/registration/OidcClientRegistrationConfig.java

```diff
@@ -0,0 +1,70 @@
+package io.quarkus.oidc.client.registration;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.oidc.common.runtime.OidcCommonConfig;
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+//https://datatracker.ietf.org/doc/html/rfc7592
+//https://openid.net/specs/openid-connect-registration-1_0.html
+
+@ConfigGroup
+public class OidcClientRegistrationConfig extends OidcCommonConfig {
```

Comment:
Use the interface pattern here too (https://quarkus.io/guides/writing-extensions#configuration)

Reply:
We'll need to have #39185 first but if this PR happens to go in earlier than this task will be done as part of #39185. I'm not sure what is the right course of action here.